tap-snapshots/test-run-flow.js-TAP.test.js

```diff
@@ -0,0 +1,35 @@
+/* IMPORTANT
+ * This snapshot file is auto-generated, but designed for humans.
+ * It should be checked into source control and tracked carefully.
+ * Re-generate by setting TAP_SNAPSHOT=1 and running tests.
+ * Make sure to inspect the output below.  Do not ignore changes!
+ */
+'use strict'
+exports[`test/run/flow.js TAP flow > must match snapshot 1`] = `
+TAP version 13
+ok 1 - cli-tests/flow/ok.js # SKIP no tests found {
+    1..0 # no tests found
+}
+
+not ok 2 - cli-tests/flow/ok.js # SKIP no tests found {
+    1..0 # no tests found
```

Comment:
I think this is the double-spawn I mentioned above.  Note that the test runs twice!

It seems weird that it's saying `1..0 # no tests found`.  Shouldn't it still run the test file?

Reply:
Fixed